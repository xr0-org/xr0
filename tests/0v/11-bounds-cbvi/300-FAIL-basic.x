#ifdef XR0

int
index() ~ [ return [0?50]; ];

#endif

int
index()
{	
	return 2;
}

int
main()
{
	int a[3];
	int i;
	int res;
	a[0] = 4;
	a[1] = 5;
	a[2] = 6;

	i = index();
	res = a[i];	/* ERROR: `i' could be out of bounds */
}
