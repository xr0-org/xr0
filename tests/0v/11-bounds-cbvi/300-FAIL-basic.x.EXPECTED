11-bounds-cbvi/300-FAIL-basic.x:18:12: out of bounds (main)
